lessons/operator/02/README.md

```diff
@@ -1,257 +1,267 @@
+   - A chat-ops environment, for alerting:
+     - `rocketchat`: [Rocket.Chat](https://rocket.chat/), a web-based chat app similar to Slack
+     - `mongo`: [MongoDB](https://www.mongodb.com/), Rocket.Chat's database
+   - A telemetry stack:
+     - `influxdb`: [InfluxDB](https://www.influxdata.com/) for metrics storage
+     - `grafana`: [Grafana](https://grafana.com/) for visualization
+   - Some support utilites:
+     - `vault`: A [HashiCorp Vault](https://www.vaultproject.io/) server, for secrets management
+     - `configurator`: A simple configuration management tool built for this workshop
+
+## How It All Works Together
+
+The basic flow of information in this environment looks something like this:
+- The _agent_ container simulates a web application node, with the Sensu agent running on it. 
+- The agent observes the application and generates _events_, which are sent to the _backend_ container.
+- The backend will make decisions about the _event_ based on the user defined configuration.
+
+Depending on the configuration, and the details of the event, the backend may raise _alerts_ which are sent to _Rocket.Chat_, or send _events_ containing _metrics_ to _InfluxDb_ to be recorded. The _Grafana_ dashboard visualizes the metrics stored in InfluxDB.
+
+Access to these third-party services requires authentication. 
+Secrets are securely delivered through _HashiCorp Vault_ directly to the backend.
+
+The _configurator_ runs only during the initial setup of the workshop, to wire everything together and apply the configuration.
+
+## EXERCISE 1: Install the Workshop
+### Scenario
+
+You are new to Sensu and are using the Sensu Workshop.
+You want to mimic a production Sensu environment, so that you can experiment with Sensu concepts and workflows.
+
+### Solution
+
+To accomplish this, we will setup a local sandbox environment using `docker-compose`.
+Each component will be created within a container running on your local machine.
+The installation and configuration is automated for you, so you can start using Sensu right away. 
+
+### Steps
+> **BEFORE YOU START:** Make sure that you've looked over all the [prerequisites](#prerequisites), and that Docker is running.
+
+1. **Clone the Workshop Repository.**
+
+   ```shell
+   git clone https://github.com/sensu/sensu-go-workshop.git
+   cd sensu-go-workshop
+   ```
+
+1. **Docker Compose Initialization.**
+
+   Create the workshop environment with the `docker-compose up` command.
+
+   **Mac and Linux**:
+   
+   ```shell
+   sudo docker-compose up -d
+   ```
+
+   **Windows**:
+   ```shell
+   docker-compose up -d
+   ```
+
+   > **NOTE:** On Linux/macOS `sudo` is required because Docker runs as root. 
+   > If you'd like to configure Docker to not need `sudo`, read the [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) documentation.
+
+   **Example Output:**
```

Comment:
This is an old example - we should update it with the actual docker-compose-default.yaml output.